### PR TITLE
DOCS-3052 iOS Monitoring Edits

### PR DESCRIPTION
**Pull request by alai97**
### What and why?

A short description of what changes this PR introduces and why.

### How?

Implements absolute URLs for the doc site and copy nits.

### Review checklist

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration)
- [x] Make sure each commit and the PR mention the Issue number or JIRA reference

### CODE REVIEW

**Review by apigirl — APPROVED** [review]

File: docs/rum_collection/advanced_configuration.md [hunk 1]
```diff
@@ -106,18 +105,17 @@ Find more details and available options in `DDRUMMonitor` class.
...
 Global.rum.addError(message: "error message.")
 ```
 
 For more details and available options, refer to the code documentation comments in `DDRUMMonitor` class.
 
-
 ## Track custom global attributes
 
-In addition to the [default RUM attributes][11] captured by the mobile SDK automatically, you can choose to add additional contextual information, such as custom attributes, to your RUM events to enrich your observability within Datadog. 
+In addition to the [default RUM attributes][6] captured by the mobile SDK automatically, you can choose to add additional contextual information, such as custom attributes, to your RUM events to enrich your observability within Datadog.
```
Comment (ncreated) [inline docs/rum_collection/advanced_configuration.md:118]: I think the link is broken, should be:
```suggestion
In addition to the [default RUM attributes][7] captured by the mobile SDK automatically, you can choose to add additional contextual information, such as custom attributes, to your RUM events to enrich your observability within Datadog. 
```

Changes to links in this PR:
```diff
- [11]: /real_user_monitoring/ios/data_collected?tab=session#default-attributes
+ [6]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces?tab=browserrum
+ [7]: https://docs.datadoghq.com/real_user_monitoring/ios/data_collected?tab=session#default-attributes
```

File: docs/rum_collection/advanced_configuration.md [hunk 2]
```diff
@@ -290,24 +294,26 @@ let session = URLSession(
     delegateQueue: nil
 )
 ```
-This tracks all requests sent with the instrumented `session`. Requests matching the `example.com` domain are marked as "first party" and tracing information is sent to your backend to [connect the RUM resource with its Trace][10].
+
+This tracks all requests sent with the instrumented `session`. Requests matching the `example.com` domain are marked as "first party" and tracing information is sent to your backend to [connect the RUM resource with its Trace][7].
```
Comment (ncreated) [inline docs/rum_collection/advanced_configuration.md:298]: The link seems broken, it should be IMO:
```suggestion
This tracks all requests sent with the instrumented `session`. Requests matching the `example.com` domain are marked as "first party" and tracing information is sent to your backend to [connect the RUM resource with its Trace][6].
```

Changes to links in this PR:
```diff
- [10]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces?tab=browserrum
+ [6]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces?tab=browserrum
+ [7]: https://docs.datadoghq.com/real_user_monitoring/ios/data_collected?tab=session#default-attributes
```

**Review by ncreated — CHANGES_REQUESTED** [review]
Thanks a lot 🚀💪! I left two feedbacks on (potentially) broken links.

**Review by ncreated — APPROVED** [review]
🚀 Thanks, I appreciate all the help on docs!

**Review by xgouchet — APPROVED** [review]

**Pull request merged**